REPORTING.md

```diff
@@ -204,4 +211,59 @@ Report on interchanges that have not received RECEP from `2020-01-01` to `2020-0
                 {$or: [{"recepCode": {"$exists":false}}, {"recepCode": null}, {"recepDateTime": {"$exists":false}}, {"recepDateTime" :null}]}
             ]
         }
-    )
\ No newline at end of file
+    )
+    
+### Interchange-OperationId Report
+
+To report on all OperationIds that correspond to a range of interchange sequence numbers:
+
+    db.getCollection('outboundState').find(
+        {
+            $and: [
+                {intSeq: { $gte : <from_interchange_sequence_number>} },
+                {intSeq: { $lte : <to_interchange_sequence_number>} },
+            ]
+        },
+        {
+            intSeq: 1,
+            operationId: 1,
+            _id: 0
+        }
+    )
+    
+where
+
+- `<from_interchange_sequence_number>` is the first sequence number in the range NHAIS reported as lost
+- `<to_interchange_sequence_number>` is the last sequence number in the range NHAIS reported as lost
+
+### Interchange-OperationId Report Example
+
+Searching for OperationIds for interchanges 3-10:
+
+    db.getCollection('outboundState').find(
+            {
+                $and: [
```

Comment:
Need to add 'sndr' and 'recip' filters because these will apply to a single pair